.github/dependabot.yml

```diff
@@ -10,6 +10,3 @@ updates:
     labels:
       - "npm"
       - "dependencies"
-    commit-message:
-      prefix: ":arrow_up: build"
```

Comment:
Ooh can we keep this actually? And just change it to the direct emoji, ⬆, rather than `:arrow_up:`? I do like the emojis in the PR titles 😄

Reply:
Ah that's true.. Let it keep looking pretty 😉